.github/pr-comments/for-labeller.md

```diff
@@ -0,0 +1,9 @@
+:smiley_cat: Thank you for creating this PR! To publish your content to [Zowe Docs](https://docs.zowe.org/), follow these required steps.
+
+- [ ] Add the label `review: doc`.
+- [ ] Identify your content topic with a label. (Examples: `area: apiml`, `area: cli`, `area: install and config`, etc.)
+- [ ] Specify the major Zowe release for your content. (Examples: `release: V1`, `release: V2`, `release: V3`)
```

Comment:
As discussed, for v3, it seems like a good idea to inform the PR opener to target the v3 branch. Glad we discussed this. ;-)